<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarReservaWindow</class>
 <widget class="QDialog" name="AgregarReservaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>313</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Ingreso del Reserva&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;CÃ©dula Cliente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCedula">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Tipo de Espacio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTipos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Espacios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEspacios">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Fecha Inicial&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarInicial">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Fecha Final&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarFinal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>570</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>570</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Facturar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Precio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Estado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
